### PR TITLE
[dnm]Update banned content to include open source bad actors

### PR DESCRIPTION
**Pull request by MrsTonedOne**
# This is a meme pr, do not merge.

**Kite** goes and hires creators of popular open source software and then convinces those employees to inject promotional shit into that open source software.

https://theoutline.com/post/1953/how-a-vc-funded-company-is-undermining-the-open-source-community

This makes it clear that they aren't welcomed here.

https://github.com/autocomplete-python/autocomplete-python/issues/308

https://github.com/atom-minimap/minimap/issues/588

### CODE REVIEW

File: .github/CONTRIBUTING.md [hunk 1]
```diff
@@ -363,9 +363,12 @@ There is no strict process when it comes to merging pull requests. Pull requests
 
 ## Banned content
 Do not add any of the following in a Pull Request or risk getting the PR closed:
+* Code that interacts with, sends data to, receives data from, or other wise has anything to do with Kite, Kite.com, or any affiliate companies.
 * National Socialist Party of Germany content, National Socialist Party of Germany related content, or National Socialist Party of Germany references
+* Code, Pull Requests, or Issue Reports submitted by Kite, Kite.com, or their employees or affiliates
```
Comment (Supermichael777) [inline .github/CONTRIBUTING.md]: whom are acting as agents of Kite or Kite.com or otherwise advancing the interests of Kite or Kite.com.

**Comment by Supermichael777** [comment]
Re-host as truly open license software MSO.

bring your biased heroism to the whole intewebz

**Comment by pubby** [comment]
MSO is higher than Kite

File: .github/CONTRIBUTING.md (hunk 1 above)
Comment (MrStonedOne) [inline .github/CONTRIBUTING.md]: No, the point is to make it clear that kite employees are not welcomed here, regardless of intent.

It also greenlights mass removing any code by anybody who later becomes an employee of kite, to reduce their stake in this repo.

Comment (DoctoreMad) [inline .github/CONTRIBUTING.md]: >It also greenlights mass removing any code by anybody who later becomes an employee of kite, to reduce their stake in this repo.

Okay so if a coder gets a silly spite towards your codebase he can announce he joined kite and watch as you start eating parts of the codebase in a f~~it of rage~~ crusade of righteousness?

**Comment by DoctoreMad** [comment]
So what is the point of this again? Are you placing bait for "Kite employees" or something?

**Comment by MrStonedOne** [comment]
It's a meme pr.

Its here to meme.

Whats not to understand?

File: .github/CONTRIBUTING.md (hunk 1 above)
Comment (MrStonedOne) [inline .github/CONTRIBUTING.md]: I think you missed the part where this is a meme pr that will never get merged.

**Comment by JJRcop** [comment]
[Congrats on the last word.](https://github.com/atom-minimap/minimap/issues/588#issuecomment-317547010)

**Comment by octareenroon91** [comment]
Is this PR going to close at some point?

What kind of meme is this

**Comment by Buggy123** [comment]
As a employee of Kite™, I see no reason to add this PR. Instead, might I suggest signing up for our Kiteport™ package, one of the many Kitervices™ we offer here at Kite™? With Kiteport™, our Kitevelopers™ will encrypt your codebase, ensuring only KiteQuality™ PRs are created, improving your software quality by Kitefold™! V™i™s™i™t™ K™i™t™e™.™c™o™m™ f™o™r™ d™e™t™a™i™l™s™.™™

File: .github/CONTRIBUTING.md [hunk 2]
```diff
@@ -363,9 +363,12 @@ There is no strict process when it comes to merging pull requests. Pull requests
...
 * Code where one line of code is split across mutiple lines (except for multiple, separate strings and comments; in those cases, existing longer lines must not be split up)
 
+
```
Comment (lordpidey) [inline .github/CONTRIBUTING.md]: Undocumented change in PR body, you made no mention of changing the spacing between points in documentation

**Review by lordpidey — CHANGES_REQUESTED** [review]

**Pull request closed without merging**